dis_snek/client/client.py

```diff
@@ -1831,6 +1832,27 @@ async def fetch_voice_regions(self) -> List["VoiceRegion"]:
         regions = VoiceRegion.from_list(regions_data)
         return regions
 
+    async def connect_to_vc(
+        self, guild_id: "Snowflake_Type", channel_id: "Snowflake_Type", muted: bool = False, deafened: bool = False
+    ) -> ActiveVoiceState:
+        """
+        Connect the bot to a voice channel.
+
+        Args:
+            guild_id: id of the guild the voice channel is in.
+            channel_id: id of the voice channel client wants to join.
+            muted: Whether the bot should be muted when connected.
+            deafened: Whether the bot should be deafened when connected.
+
+        Returns:
+            The new active voice state on successfully connection.
+
+        """
+        return await self._connection_state.voice_connect(guild_id, channel_id, muted, deafened)
+
+    def get_bot_voice_state(self, guild_id: "Snowflake_Type") -> ActiveVoiceState:
```

Comment:
Wrong return annotation
```suggestion
    def get_bot_voice_state(self, guild_id: "Snowflake_Type") -> Optional[ActiveVoiceState]:
```